solr/contrib/ltr/example/train_and_upload_demo_model.py

```diff
@@ -10,8 +10,8 @@
 
 solrQueryUrl = ""
 
-def generateQueries(config):
-        with open(config["userQueriesFile"]) as input:
+def generateQueries(userQueriesFile, config):
+        with open(userQueriesFile) as input:
```

Comment:
why pass in userQueriesFile since you already pass in config with it?  You already removed all the other references to config for the other functions